daemon/cluster/tasks.go

```diff
@@ -45,6 +45,11 @@ func (c *Cluster) GetTasks(options apitypes.TaskListOptions) ([]types.Task, erro
 		return nil
 	}
 
+	if len(options.Filters.Get("runtime")) == 0 {
+		options.Filters.Add("runtime", "container")
+		options.Filters.Add("runtime", "")
```

Comment:
Let's make a copy of the `options.Filters` instead of modifying the filters map that the caller passes in.